crowbar_framework/lib/crowbar/repository.rb

```diff
@@ -138,16 +134,25 @@ def bag_item
 
     private
 
+    def repos_dir
```

Comment:
To be in same naming... `repos_path` or change the method below to `repodata_dir`.